packages/OcLaunch/OcLaunch.0.1.3/descr

```diff
@@ -0,0 +1,6 @@
+Launch commands automatically
+OcLaunch is a command-line tool to launch successively (each time the
+program is called) commands. It is designed to be used with any
+program, interactive or not. This a early version. Feedback is
+welcomed at leowzukw@vmail.me.
```

Comment:
The `opam` file already contains your email.  Not sure duplicating it here is needed.